airbyte-integrations/connectors/source-confluence/source_confluence/manifest.yaml

```diff
@@ -41,13 +41,21 @@ definitions:
 
   requester:
     type: HttpRequester
-    url_base: https://{{ config['domain_name'] }}/wiki/rest/api/
+    url_base: "{{ config['base_url'] + '/rest/api/' if config.confluence_server else config['base_url'] + '/wiki/rest/api/' }}"
```

Comment:
```suggestion
    url_base: "{{ "https://" + config['domain_name'] + '/rest/api/' if config.confluence_server else "https:// + " + config['domain_name'] + '/wiki/rest/api/' }}"
```

Reply:
This way it doesn't introduce a breaking change to existing connections and allow you to use the new option.

Reply:
Done